### ISSUE TITLE
When creating an alert, we should auto-select the first valid channel

### ISSUE THREAD

**Issue by luizarakaki** [metabase/metabase#48224]
### Describe the bug

![image](https://github.com/user-attachments/assets/5dabe5cd-a20c-4513-b172-c9eb2f920834)

We should select Slack instead of not-configured Email

### To Reproduce

1. Add Slack or Webhook and don't add an Email server
2. Try to set up an alert
3. See that it is selecting Email

### Expected behavior

Select the first valid channel

### Information about your Metabase installation

```JSON
master 012408e
```

### Severity

P3

**Comment by cdeweyx** [comment]
I think this is .Workflows? cc: @KorowkaMu feel free to correct me if I'm wrong

**Comment by KorowkaMu** [comment]
@cdeweyx yes, this one is .Workflows, thank you for tagging. It will be fixed with the new alerts UI.
